client/src/app.jsx

```diff
@@ -13,13 +13,11 @@ class App extends React.Component {
       paused: true, // change to false if i want to autoplay on load
       trackNumber: 0,
       displayList: false,
-      displayVolume: false
+      displayVolume: false,
+      seekValue: 0
     };
```

Comment:
Just a suggestion: maybe consider binding all your functions here in your constructor?  Easier to keep track what's been binded (bound?)

Reply:
make it hell for them

Reply:
but will look into if i finish the project first